var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -50,7 +52,7 @@ class Cuda(Package):
             url="http://developer.download.nvidia.com/compute/cuda/6_5/rel/installers/cuda_6.5.14_linux_64.run")
 
     def install(self, spec, prefix):
-        runfile = glob(join_path(self.stage.path, 'cuda*run'))[0]
```

Comment:
was it a bug or shall this be version dependent?

Reply:
But then this line must be conditional on the version

Reply:
don't worry, the modified regex fits both old releases and the latest one that changed the file name :)